Task/Tasks/TSKTask.m

```diff
@@ -404,6 +406,11 @@ - (void)cancel
 }
 
 
+- (void)didCancel
+{
```

Comment:
You don't want an `Intentionally left blank` comment on these?

![left blank](http://familypcrepair.com/wp-content/uploads/2014/10/Left-blank.jpg)